src/utils/helpers/date/date.js

```diff
@@ -32,8 +30,14 @@ const DateHelper = {
    * @param {Object} options Override Moment JS options
    * @return {Boolean}
    */
-  isValidDate: (value, options = {}) => {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks